invenio_rdm_records/services/schemas/record.py

```diff
@@ -65,6 +65,7 @@ class Meta:
     revision = fields.Integer(dump_only=True)
     versions = NestedAttribute(VersionsSchema, dump_only=True)
     parent = NestedAttribute(RDMParentSchema)
+    is_published = fields.Boolean(dump_only=True)
```

Comment:
adding it back, suggested to do by @max-moser

Reply:
yes, having a look at my original [record deletion PR](https://github.com/inveniosoftware/invenio-rdm-records/pull/1386/files#diff-3fc71b0ed029b66a27f137d5f3367a0f066cbc0068aba9ca19048169d017dd10L68-R68) I'm relatively confident that this was an accidental rename on my side